talks/_posts/2017-11-03-intro-to-distributed-computing.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Intro to Distributed Computing
+author: Christian Martin
+layout: post
+slides: /talks/2017-w10-fall-intro-to-distributed computing.pdf
```

Comment:
Adding a review to request changes, just so this PR isn't merged by accident until the slides are added. 😃